scipy/stats/contingency.py

```diff
@@ -250,17 +266,19 @@ def chi2_contingency(observed, correction=True, lambda_=None):
     ...       [[14, 17],
     ...        [15, 16]]]])
     >>> chi2_contingency(obs)
-    (8.7584514426741897,
-     0.64417725029295503,
-     11,
-     array([[[[ 14.15462386,  14.15462386],
-              [ 16.49423111,  16.49423111]],
-             [[ 11.2461395 ,  11.2461395 ],
-              [ 13.10500554,  13.10500554]]],
-            [[[ 19.5591166 ,  19.5591166 ],
-              [ 22.79202844,  22.79202844]],
-             [[ 15.54012004,  15.54012004],
-              [ 18.10873492,  18.10873492]]]]))
+    Chi2ContingencyResult(
```

Comment:
This is problematic for our ref guide check. I would suggest to not print the whole object at once to make this readable.